setup.py

```diff
@@ -22,7 +23,7 @@
 setup_requires = ['pytest-runner'] if {'pytest', 'test', 'ptr'}.intersection(sys.argv) else []
 
 setuptools.setup(
-    name='ark-crypto',
```

Comment:
This name is just the name that'll be used in pypi and requirements.txt. I'd leave it as `ark-crypto` as that makes it unique on pypi. The folder name (module) can be left `crypto` so you'll then use it by `from crypto.blah import blah` but when installing you'll do `pip install ark-crypto` to avoid confusion and pypi conflicts.